.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
```

Comment:
I'm only building hieroplatform here and running its tests. I think users are having trouble setting up tomcat.